components/network/vpnc/Makefile

```diff
@@ -15,21 +15,26 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		vpnc
-COMPONENT_VERSION=	0.5.3
```

Comment:
Can we preserve COMPONENT_VERSION, bump COMPONENT_REVISION and set HUMAN_VERSION to something like $(COMPONENT_VERSION).r550 ?